Tests/Fixtures/Generated/Colors/swift4/defaults-customName.swift

```diff
@@ -35,18 +35,35 @@ internal struct XCTColors {
 
 // MARK: - Implementation Details
 
-// swiftlint:disable operator_usage_whitespace colon
 internal extension XCTColor {
   convenience init(rgbaValue: UInt32) {
-    let red:   CGFloat = CGFloat((rgbaValue >> UInt32(24)) & UInt32(0xff)) / CGFloat(255.0)
-    let green: CGFloat = CGFloat((rgbaValue >> UInt32(16)) & UInt32(0xff)) / CGFloat(255.0)
-    let blue:  CGFloat = CGFloat((rgbaValue >> UInt32( 8)) & UInt32(0xff)) / CGFloat(255.0)
-    let alpha: CGFloat = CGFloat((rgbaValue              ) & UInt32(0xff)) / CGFloat(255.0)
+    let components = RGBAComponents(rgbaValue: rgbaValue).normalized
+    self.init(red: components[0], green: components[1], blue: components[2], alpha: components[3])
+  }
+}
+
+fileprivate struct RGBAComponents {
```

Comment:
Seems this generates a SwiftLint violation 🤔 

```
 warning: Private over fileprivate Violation: Prefer `private` over `fileprivate` declarations
```

Which is a common SwiftLint warning/rule… in that case I'm not sure tbh: I _think_ we _do_ indeed want `fileprivate`, but the SwiftLint rule says that we usually always want `private` over `fileprivate` for top level definition? (I have to refresh a bit my knowledge of the difference between the two in that special case to be sure…)

Reply:
AFAIK, a `private` extension is visible for other types in the same file, so that'd work here. `fileprivate` annotations would only be needed here for specific functions/properties.

Reply:
@djbe That's not an `extension` here though, that's the declaration of a new `struct` 😉 
So I think making it `private struct` would still expose that `struct` to any other files in the same module, right?

Reply:
Ah yeah, it's a `struct`. Shouldn't matter, `private` type definitions don't go beyond the file. `private` properties/functions on the other hand don't go beyond the Type.